sycl/include/CL/sycl/accessor2.hpp

```diff
@@ -458,9 +458,9 @@ class accessor :
 
   constexpr bool is_placeholder() const { return IsPlaceH; }
 
-  size_t get_size() const { return getRange().size() * sizeof(DataT); }
+  size_t get_size() const { return getOrigRange().size() * sizeof(DataT); }
```

Comment:
`getOrigRange` is very unfortunate name.
Could you rename to something like `getMemoryObjectRange`, please?

`getRange` name should also be improved to avoid ambiguity - `getAccessRange`.

Reply:
Good idea, I am in process of doing additional changes for this. 
BTW, how about having getMemoryRange() instead of getMemoryObjectRange(). The second is a bit more clear, but it a bit too long. Also, AccessorImplDevice and LocalAccessorBaseDevice already have the fields 'Range' and 'MemRange'. 
I think renaming the field 'Range' to 'AccessRange' and keeping 'MemRange' field would be sufficient there.

Reply:
I'm fine with getMemoryRange too. Another option: getAllocatedRange.
All of them much more specific than just getRange. :-)

Reply:
Fixed. The updated patch will be uploaded to here in a minute.